### DISCUSSION TITLE
PLUGINS IN SERVER BUT NOT IN DAW

### DISCUSSION THREAD

**Discussion by Dumdididum** [apohl79/audiogridder#1133] [General]
On the Server side, it list the plugins that are "working" and at the bottom or the list, it shows the plugins that do not work with Audiogridder. 
When I load an AG plugin into my DAW, it of course will not show the non working plugins, but it seems to be missing a lot of the plugins that are on the "working" list on the Server as well. 
It does come up with some of the plugins, but not all of them.

**Comment by apohl79** [comment]
Please follow https://audiogridder.com/bug-reports/
